core/controllers/topic_editor.py

```diff
@@ -79,11 +78,6 @@ def post(self, topic_id):
         topic_domain.Topic.require_valid_topic_id(topic_id)
         title = self.payload.get('title')
 
-        topic = topic_services.get_topic_by_id(topic_id, strict=False)
```

Comment:
Why was this removed? 
Granted `acl_decorator` checks if topic rights is present, but it and the topic itself may get disjoined, so if at all you are removing this from here, add check for topic also to the decorator.

Reply:
Done

Reply:
Reverted the changes as its gonna make the next few lines untested

Reply:
Reverted which changes?

Reply:
Reverted the changes in acl_decorators to check if topic is None. This will make the line with `get_topic_rights` untested.

Reply:
If you are reverting this, please add the check for topic in addition to topic rights in `acl_decorators` as I mentioned earlier.

Reply:
Also, as mentioned in other PRs, replace this check with a comment explaining why we can be confident it's safe to use here.

Reply:
Done

Reply:
Where's the comment?